src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/sphinx-design.css

```diff
@@ -1,152 +1,227 @@
 @import "../design-style.4045f2051d55cab465a707391d5b2007.min.css";
-/*
-##############
-Sphinx design
-##############
-*/
-
-blockquote {
-  background-color: var(--pst-color-background);
-}
-.sd-sphinx-override,
-.sd-sphinx-override * {
-  flex: auto;
-}
 
-/* Sphinx-design tab */
+/**
+sphinx design button
+*/
 
-/* Common styles for all screen sizes */
-.sd-tab-set > input:not(.focus-visible) + label {
-  outline: none;
-  font-size: large;
-  -webkit-tap-highlight-color: var(--pst-color-background) !important;
-  color: var(--pst-color-text-base);
+.sd-btn {
+  font-size: var(--ast-sphinx-design-font-size);
+  font-weight: 400;
+  font-family: var(--ast-sphinx-design-font-family);
+  padding: 8px 16px;
+  border-radius: var(--ast-sphinx-design-border-radius);
 }
 
-.sd-tab-set > input:checked + label + .sd-tab-content {
-  display: block;
-  font-size: medium;
+.sd-btn-secondary {
+  font-size: var(--ast-sphinx-design-font-size);
+  font-weight: 400;
+  font-family: var(--ast-sphinx-design-font-family);
+  padding: 8px 16px;
+  border-radius: var(--ast-sphinx-design-border-radius);
+  color: var(--sd-color-secondary-text);
+  background-color: var(--sd-color-secondary);
+  border-color: var(--sd-color-secondary);
 }
 
-.sd-tab-set > input:checked + label + .sd-tab-content {
-  display: block;
-  font-size: 0.9rem;
+blockquote {
+  background-color: var(--pst-color-background);
 }
 
-.sd-tab-content {
-  font-family: var(--pst-font-family-base);
+html[data-theme="dark"] .sd-shadow-sm {
+  box-shadow: 0 0.1rem 1rem rgba(250, 250, 250, 0.6) !important;
 }
```

Comment:
This shadow does not fit with the rest of the theme. Also, shadows only exist in light themes, see the effect in PyData Sphinx Theme: https://pydata-sphinx-theme.readthedocs.io/en/stable/

In light mode, they project the shadow of the element. In dark mode, there is no shadow but the component displays a thin solid white border.

How should we proceed, @mia-guo-ux?

Reply:
In dark mode, the shadow should be 
_box-shadow: 0px 1px 2px 0px, rgba(0, 0, 0, 0.40), 0px 2px 4px 0px, rgba(0, 0, 0, 0.48)_
![image](https://github.com/user-attachments/assets/cc6b34f9-1ec2-4db8-956e-47d4281b90f0)

Btw do you have access to Figma? There is a Dev Mode in Figma, which shows the specs and codes.
![image](https://github.com/user-attachments/assets/7a0ce8f9-6901-42eb-85fe-6aa9801bcf94)

Reply:
Updated the dark box shadow. about figma,  i dont have access to the Figma.

Reply:
> Updated the dark box shadow. about figma, i dont have access to the Figma.

Looking good! Can we also implement a border radius of 8px to cards in dark theme?  Thanks!

Reply:
sure, i am adding it.

Reply:
border radius added 
![image](https://github.com/user-attachments/assets/d0ec0c7e-be0a-4435-abde-9b5e4ffeecd9)

Reply:
Nice!